sid/golang/assignment1/handlers/handlers.go

```diff
@@ -0,0 +1,106 @@
+package handlers
+
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/farkaskid/go_assignments/assignment1/helpers"
+)
+
+func Handle(cmd string, history *helpers.History) int {
+	// Main handler for commands, this will branch off to different handlers for different commands
+
+	commandFields := strings.Fields(cmd)
+
+	switch commandFields[0] {
+
+	case "ls":
+		return lsHandler(commandFields[1:])
+
+	case "cd":
+		return cdHander(commandFields[1:])
+
+	case "pwd":
+		return pwdHandler(commandFields[1:])
+
+	case "history":
+		return historyHandler(history)
+
+	default:
+		fmt.Println("Unknowm command:", cmd)
+		return 1
+	}
+}
+
+func lsHandler(inputs []string) int {
```

Comment:
Any reason for returning `int` from all helper funcs? Also this return value is not handled in the main parent func. If required follow go idiomatic way of returning errors instead of int